.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,22 @@
 #### Which issue(s) does this change fix?
 <!-- Use the format #<issue-number>, e.g. #42 -->
 
+
 #### Why is this change necessary?
 
+
 #### How does it address the issue?
 
+
 #### What side effects does this change have?
 
-#### Checklist
+
+#### Mandatory Checklist
+**PRs will only be reviewed after checklist is complete**
 
 - [ ] Add input/output [type hints](https://docs.python.org/3/library/typing.html) to new functions/methods
-- [ ] Write design document ([Do I need to write a design document?](https://github.com/aws/aws-sam-cli/blob/develop/DEVELOPMENT_GUIDE.md#design-document))
-- [ ] Write unit tests
+- [ ] Write design document if needed ([Do I need to write a design document?](https://github.com/aws/aws-sam-cli/blob/develop/DEVELOPMENT_GUIDE.md#design-document))
+- [ ] Write/update unit tests
 - [ ] Write/update functional tests
```

Comment:
I feel that for SAMCLI functional tests are more or less related to validate only so maybe we can add a condition here?